src/Components/CreateImageWizard/steps/fileSystemConfiguration.js

```diff
@@ -125,10 +125,7 @@ const fileSystemConfigurationStep = {
         </TextContent>
       ),
       condition: {
-        or: [
-          { when: 'file-system-config-radio', is: 'automatic' },
-          { when: 'oscap-profile', is: undefined },
-        ],
+        or: [{ when: 'file-system-config-radio', is: 'automatic' }],
```

Comment:
We could also remove the `or` as there is only one condition now.

```suggestion
        when: 'file-system-config-radio',
        is: 'automatic',
```